core/types.go

```diff
@@ -19,6 +19,9 @@ type Result struct {
 	// HTTPStatus is the HTTP response status code
 	HTTPStatus int `json:"status"`
 
+	//
+	DNSRCode string `json:"dnsr_code"`
```

Comment:
Can you add the missing comment and change `json:"dnsr_code"` to `json:"dns-rcode"`?